jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -196,9 +196,10 @@ The CircleCI Amazon SageMaker orb requires some environment variables to functio
 * `CCI_RELEASE_INTEGRATION_TOKEN`: The orb connects your deployment to SageMaker with xref:deploy/deploys-overview#[CircleCI deploys]. This gives you visibility into the Endpoint Configuration Updates, and what is currently active.
 * `SAGEMAKER_EXECUTION_ROLE_ARN`: This is the AWS IAM Role you configured with the necessary SageMaker permissions, and the OIDC Trust relationship.
 
-. From the CircleCI web app, click **Dashboard** in the sidebar.
-. Use the project filter dropp-down to choose your project.
-. Select btn:[Project Settings] and then choose **Environment Variables**.
+include::../_includes/partials/app-navigation/steps-to-project-settings.adoc[]
```

Comment:
The formatting is off in my local build. It's putting the first 3 steps as sub steps for the preceding bullet 😕 
If you add a line in after the bullets that fixes the formatting...